templates/ovs_stats.conf.j2

```diff
@@ -1,8 +1,16 @@
 LoadPlugin ovs_stats
 
 <Plugin ovs_stats>
-#  Port "6640"
-#  Address "127.0.0.1"
-#  Socket "/var/run/openvswitch/db.sock"
-#  Bridges "br0" "br_ext"
+{% if collectd_plugin_ovs_stats_port is defined %}
+  Port "{{ collectd_plugin_ovs_stats_port }}"
+{% endif %}
+{% if collectd_plugin_ovs_stats_address is defined %}
+  Address "{{ collectd_plugin_ovs_stats_address }}"
+{% endif %}
+{% if collectd_plugin_ovs_stats_socket is defined %}
+  Socket "/var/run/openvswitch/db.sock"
```

Comment:
This is hardcoded? Wouldn't you use a variable here?

Reply:
> Would it make sense to include options in default/...?

In this case, all the values have defaults set by collectd itself, so I didn't want to overrride them, as it seems unintuitive. I've been trying to only set defaults when a value is mandatory, since you can't unset a variable in ansible, and I don't want users to be forced to set unnecessary variables.
Because all the config values are optional, I would be left with a functionally empty defaults/ovs_stats.yml file, which would cause errors in the rendering, since data files need to have content.

The values that were previously hardcoded came from either Barometer or TripleO, and I plan to add defaults for those values into the tripleo-collectd config.

Reply:
> This is hardcoded? Wouldn't you use a variable here?

This is a mistake on my part, thanks for spotting!